codebase/gatling-tests/src/main/resources/application.conf

```diff
@@ -1,5 +1,11 @@
 api.version="v1"
-server {
+identityService {
+  host = "127.0.0.1"
+  host = ${?TEST_SUBJECT_HOST}
+  port = "8100"
+  port = ${?TEST_SUBJECT_PORT}
```

Comment:
@lbialy Since we have 2 hosts defined here, TEST_SUBJECT_HOST should be changed to something more meaningful